core/drivers/scmi-msg/entry.c

```diff
@@ -88,3 +96,34 @@ void scmi_process_message(struct scmi_msg *msg)
 
 	scmi_status_response(msg, SCMI_NOT_SUPPORTED);
 }
+
+#ifdef CFG_SCMI_MSG_SMT_FASTCALL_ENTRY
+void scmi_smt_fastcall_smc_entry(unsigned int channel_id)
+{
+	static uint32_t payload[CFG_TEE_CORE_NB_CORE][SCMI_PAYLOAD_U32_MAX];
+
+	assert(!plat_scmi_get_channel(channel_id)->threaded);
+
+	scmi_entry_smt(channel_id, payload[get_core_pos()]);
+}
+#endif
+
+#ifdef CFG_SCMI_MSG_SMT_INTERRUPT_ENTRY
+void scmi_smt_interrupt_entry(unsigned int channel_id)
+{
+	static uint32_t payload[CFG_TEE_CORE_NB_CORE][SCMI_PAYLOAD_U32_MAX];
```

Comment:
How about moving this next to `threaded_payload` above?

Reply:
ok